src/App.tsx

```diff
@@ -1,16 +1,24 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState([...moviesFromServer]);
```

Comment:
You don't need to use spread operator there
```suggestion
  const [movies, setMovies] = useState(moviesFromServer);
```

Reply:
@Anton-Tretiak Good point.
@kateryna-babina Fix it.